actions/chart_uninstall_handler_test.go

```diff
@@ -0,0 +1,79 @@
+package actions
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+
+	"github.com/castai/cluster-controller/castai"
+	"github.com/castai/cluster-controller/helm"
+	mock_helm "github.com/castai/cluster-controller/helm/mock"
+)
+
+func TestUninstallActionValidator(t *testing.T) {
```

Comment:
It's not a good practice to test private methods. In this case you can test it via `TestChartUninstallHandler` subtest.

Reply:
Changed as suggested